modules/planning/open_space/trajectory_smoother/dual_variable_warm_start_problem.cc

```diff
@@ -57,6 +57,87 @@ bool DualVariableWarmStartProblem::Solve(
       auto t_end = cyber::Time::Now().ToSecond();
       ADEBUG << "Dual variable warm start solving time in second : "
              << t_end - t_start;
+
+      if (planner_open_space_config_.dual_variable_warm_start_config()
```

Comment:
Remove code here and wrap all checks INSIDE osqp interface.

Reply:
> Remove code here and wrap all checks INSIDE osqp interface.

I directly removed this comparison, since there is no clue ipopt is better. And I change the check_osqp flag to debug_osqp for debug constraint matrix only.